.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+	// Use IntelliSense to learn about possible attributes.
```

Comment:
Can add this use to the readme ? 

Or is it committed by mistake ?

Reply:
Done a little googling and it seems to be useful info I would keep it.